pkg/config/config.go

```diff
@@ -46,7 +82,27 @@ func GetConfig(cfgFilename string) (*Config, error) {
 		return nil, fmt.Errorf("fatal error config file: %s", err)
 	}
 	var config Config
-	err = viper.Unmarshal(&config)
+	decoderConfigOption := func(dc *mapstructure.DecoderConfig) {
```

Comment:
What's the default behavior if this value is not set ?

Reply:
since Level is an enum, it should have the default value 0, which equates to low